AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -37,7 +37,7 @@ + (void)performDbTokenAsyncOperationWithHttpClient:(id<MSHttpClientProtocol>)htt
     NSError *error =
         [[NSError alloc] initWithDomain:kMSACDataStoreErrorDomain
                                    code:MSACDataStoreInvalidPartitionError
-                               userInfo:@{NSLocalizedDescriptionKey : [NSString stringWithFormat:@"Invalid partition name %@", partition]}];
+                               userInfo:@{ NSLocalizedDescriptionKey : [NSString stringWithFormat:@"Invalid partition name %@", partition] }];
```

Comment:
@jaelim-ms manually add the space here, but seem clang-format do not want it? 🤔

Reply:
So why change it? 😉

Reply:
If the script doesn't like it, just leave it there. You don't need to manually change it.